packages/firebase_crashlytics/firebase_crashlytics/example/macos/Runner.xcodeproj/project.pbxproj

```diff
@@ -253,7 +253,6 @@
 			files = (
 				33CC10F32044A3C60003C045 /* Assets.xcassets in Resources */,
 				33CC10F62044A3C60003C045 /* MainMenu.xib in Resources */,
-				B5515A0723F6830A00F4A798 /* GoogleService-Info.plist in Resources */,
```

Comment:
or this. The file exists in the resources.

Reply:
This is the bug I'm solving, the GoogleService-Info.plist was duplicated to I removed it once from the build settings

Reply:
Ah, yeah, I see. I didn't realise there was a duplication 👍